developer/reference/articles/front-end/03-front-end-taglibs/20-liferay-theme-objects-intro.markdown

```diff
@@ -0,0 +1,30 @@
+---
+header-id: liferay-theme-objects-available-in-jsps
+---
+
+# Liferay Theme Objects Available in JSPs
```

Comment:
@shuyangzhou Can you (or a team member) review this reference on the objects available to JSPs via `>liferay-theme:defineObjects>`?

Reply:
@jhinkey It looks good to me.